lib/constants.js

```diff
@@ -2,8 +2,4 @@ export const SPARK_VERSION = '1.11.2'
 export const MAX_CAR_SIZE = 200 * 1024 * 1024 // 200 MB
 export const APPROX_ROUND_LENGTH_IN_MS = 20 * 60_000 // 20 minutes
 
-export const RPC_REQUEST = new Request('https://api.node.glif.io/', {
```

Comment:
This should be `RPC_URL` now that it's just an URL string and not a request object.

We already use that same name in other places, e.g. here:

https://github.com/filecoin-station/spark-api/blob/eea540e2086ab142b5bf9a94930fda1ec6d22032/lib/ie-contract.js#L5C8-L5C15

Reply:
https://github.com/filecoin-station/spark/pull/70